src/nxdoc/nuxeo-server/administration/database-configuration/documentdb.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Amazon DocumentDB
+labels:
+    - database
+    - documentdb
+tree_item_index: 700
+
+---
+`DocumentDB` is compatible with `MongoDB`, therefore, it can be used by `Nuxeo` like a `MongoDB` backend,
```

Comment:
Suggested text - `DocumentDB` is compatible with `MongoDB`, which allows `Nuxeo` to use `DocumentDB` as a backend for `MongoDB`.

Reply:
but DocumentDB is not a backend for MongoDB. maybe to Use DocumentDB as a backend like MongoDB.

Reply:
From a language perspective, I would suggest the following statement:
`DocumentDB` is compatible with `MongoDB`. Therefore, it can be used by `Nuxeo` as a backend database, like `MongoDB`.

Reply:
my reflection is the following counter-example:
```
postgresql can be used as a backend database, like mongodb
postgresql cannot be used like a mongodb backend (more specific and shorter)
```
so this seems to be the best fit to me:
```
documentdb can be used like/as a mongodb backend 
```